autospec/files.py

```diff
@@ -86,17 +221,15 @@ def compat_exclude(self, filename):
 
         return exclude
 
-    def file_pat_match(self, filename, pattern, package, replacement="", prefix=""):
+    def file_pat_match(self, filename, pat, package, prefix=""):
```

Comment:
Nit around keeping pattern as the variable name (though it is a pain with patterns used as a name in the same block).

Reply:
I renamed here because the local variable of the compiled object was `pat`. Do you want me to change the internal code but keep the variable name in the function declaration?

Reply:
Only because I prefer full names in function declarations though I don't think we follow that super well in autospec today.